documentation/modules/auxiliary/admin/kerberos/ticket_converter.md

```diff
@@ -0,0 +1,50 @@
+# Kerberos Ticket Converting (CCache <---> Kirbi)
+
+The `ticket_converter` module is used to convert from a ccache file format to the kirbi file format and vice versa.
```

Comment:
I think we should give the full name here, so if we embed it into the docs site it's more obvious what this means (maybe)

```
auxiliary/admin/kerberos/forge_ticket
```